db/seeds.rb

```diff
@@ -42,11 +41,28 @@
       conference_room: conference_rooms.sample,
       user: Faker::Name.name
     }
-    if (event = Event.all[event_id])
+    if (event = Event.find_by_id(event_id))
       event.update(params)
     else
       Event.create(params)
     end
     event_id += 1
   end
 end
+
+event_id = 1
+ConferenceRoom.all.each do |conference_room|
+  params = {
+    start_time: Time.now.beginning_of_day + 6.hours,
+    end_time: Time.now.beginning_of_day + 7.hours,
+    name: Faker::Company.name,
+    description: Faker::Company.catch_phrase,
+    conference_room: conference_room,
+    user: Faker::Name.name
+  }
+  if (event = Event.find_by_id(event_id))
```

Comment:
I don't get it - why we try to fetch here event with id = 1 for every single conference room